ocs_ci/ocs/perftests.py

```diff
@@ -1023,3 +1027,114 @@ def cleanup_testing_pod(self):
         except Exception:
             log.warning("The POD failed to delete")
             pass
+
+    def get_performance_metrics(self, start_time):
```

Comment:
@pintojoy does this data available for nodes? is it possible to capture memory data of all OCS nodes?